tests/testthat/test-developer-testing-rtf_rich_text.R

```diff
@@ -0,0 +1,19 @@
+test_that("rtf_rich_text fundamentally works.", {
```

Comment:
Could you give an example that works as shown in this example? 

[check_args](https://merck.github.io/r2rtf/articles/rtf-text.html#combine-text-in-different-format)

Reply:
I've added one, but it was not exactly the same due to `rtf_rich_text()` not needing to wrap every piece of text (even those without special formatting) in a `rft_text()` call.